### PR TITLE
Team/hypervisors/xen 4.12

### PR DESCRIPTION
**Pull request by idruzhinin**

### CODE REVIEW

File: xc/domain.ml [hunk 1]
```diff
@@ -254,8 +254,14 @@ let make ~xc ~xs vm_info vcpus domain_config uuid final_uuid =
     flags = flags;
     max_vcpus = vcpus;
     max_evtchn_port = -1;
```
Comment (edwintorok) [inline xc/domain.ml]: Should this be configurable as well?

**Comment by edwintorok** [comment]
This needs to go in together with the xen.pg changes

File: xc/domain.ml (hunk 1 above)
Comment (idruzhinin) [reply xc/domain.ml]: No, we only allow to configure granttable limits for now - it mirrors master commit to 4.11 fro Ross

**Comment by edwintorok** [comment]
This has been merged outside of Github when merging the xen-4.12 tag

**Pull request closed without merging**